templates/zerver/app/left_sidebar.html

```diff
@@ -63,7 +63,7 @@
                 <span class="tippy-zulip-tooltip streams_inline_cog_wrapper" data-tippy-content="{{ _('Add streams') }}" data-tippy-placement="top">
                     <i id="streams_inline_cog" class='fa fa-plus' aria-hidden="true" ></i>
                 </span>
-                <i class='streams_filter_icon fa fa-search tippy-zulip-tooltip' aria-hidden="true" data-tippy-content="{{ _('Filter streams') }} (q)"></i>
+                <i class='streams_filter_icon fa fa-filter tippy-zulip-tooltip' aria-hidden="true" data-tippy-content="{{ _('Filter streams') }} (q)"></i>
```

Comment:
We now have different icons that do the same thing, for filtering streams in the left sidebar and filtering users in the right sidebar.